arcsi/api/item.py

```diff
@@ -341,19 +349,19 @@ def download_play_file(id):
     return redirect(presigned, code=302)
 
 
-@arcsi.route("/item/<int:id>", methods=["DELETE"])
+@arcsi.route("/archon/item/<int:id>", methods=["DELETE"])
```

Comment:
Why not add `archon` route prefix for above archon APIs too?

Reply:
Same reason as below